### ISSUE TITLE
warning: already initialized constant Thread

### ISSUE THREAD

**Issue by CodeMonkeySteve** [ruby-debug/ruby-debug-ide#137]
Seeing lots of warnings about redefining `Thread` and `OldThread`, in `thread-alias/alias_thread.rb` and `thread-alias/unalias_thread.rb`.  My workaround to silence the warnings (not sure if this is the best way):

alias_thread.rb:
```
unless defined?(OldThread)
  OldThread = Thread
  self.class.send(:remove_const, 'Thread')
  Thread = Debugger::DebugThread
end
```

unalias_thread.rb:
```
if defined?(OldThread)
  self.class.send(:remove_const, 'Thread')
  Thread = OldThread
  self.class.send(:remove_const, 'OldThread')
end
```

**Comment by valich** [comment]
@CodeMonkeySteve I can't conceive a better solution than using remove_const so I'd go for it. 

I've pushed a fix with a couple of tests, and will update gem to beta3 soon. Please check it, if everything goes OK, I think it will make sense to port the fix to 0.6.1 branch.

**Comment by CodeMonkeySteve** [comment]
Your fix works great, thanks.
